pocket-archiver.js

```diff
@@ -1,27 +1,24 @@
+import * as dotenv from 'dotenv'
+dotenv.config()
 // Node.js does not support the fetch() method. To run this program in Node.js, node-fetch is installed and a package.json file with a "module" type is provided.
 import fetch from "node-fetch";
 
-// Replace ${consumer_key} with your consumer key from your Pocket application.
-const consumerKey = '${consumer_key}';
-
-//Replace ${access_token} with your access token from your Pocket application.
-const accessToken = '${access_token}';
+const {CONSUMER_KEY, ACCESS_TOKEN} = process.env
```

Comment:
typically secrets like API access tokens are loaded/accessed via environment variables using a `.env` file. Then the `.env` is added to a `.gitignore` to avoid committing to VCS and exposing them. 

The [dotenv](https://www.npmjs.com/package/dotenv) package loads values from the env file.